### PR TITLE
Convert Orguser Card to Material ui 

### LINKED ISSUE #110: Convert Orguser_info card to Material UI
Currently it has developed using ATND. Since we are moving from antd to material task is to convert the Orguser_info card section into material UI

### PR DESCRIPTION
**Pull request by ffalpha**
## Description
Converted the Org user card and org model to the materil ui. Both components are no longer depend on antd. Checked with all the existing functionality.

## Related Issue
Fix #110 

## How Has This Been Tested?
Everything is working as before 

## Screenshots or GIF (In case of UI changes):
**Click on image to see a enlarge version** 
**Antd - Previous** 

**Material ui -Current** 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by thisuraseniya** [comment]
> 

Can you change the color of the text (ADD USER) in the green button to white and check whether it is more readable than black?

File: src/components/Organization/OrgUsersCard/addOrgUserModal.js [hunk 1]
```diff
@@ -65,105 +73,90 @@ const AddOrgUserModal = ({ currentOrgHandle }) => {
...
+                  <div style={{ paddingLeft: '5px' }}>{option.name}</div>
+                </div>
+              </MenuItem>
+            ))}
+          </Select>
+        </div>
+      </Grid>
+
+      <Button
+        style={{ backgroundColor: '#0f7029' }}
```
Comment (thisuraseniya) [inline src/components/Organization/OrgUsersCard/addOrgUserModal.js:152]: I think white text would be more readable

**Review by shehand — CHANGES_REQUESTED** [review]
Just a suggestion: I saw you've used single quotes when importing and sort of other string related stuffs. Could we change it back to double quotes?

File: src/components/Organization/OrgUsersCard/addOrgUserModal.js (hunk 1 above)
Comment (ffalpha) [reply src/components/Organization/OrgUsersCard/addOrgUserModal.js:152]: will do

File: src/components/Organization/OrgUsersCard/addOrgUserModal.js [hunk 2]
```diff
@@ -1,57 +1,65 @@
...
-    </>,
-    <>
-      <SafetyOutlined className="mr-8" />
-      Admin
-    </>
+  const [handle, setHandle] = useState('');
+  const [handleValidateError, setHandleValidateError] = useState(false);
+  const [handleValidateErrorMessage, setHandleValidateErrorMessage] = useState(
+    ''
+  );
```
[the remark this answers is not in the document]
Comment (ffalpha) [reply src/components/Organization/OrgUsersCard/addOrgUserModal.js:44]: okay

**Review by thisuraseniya — APPROVED** [review]

**Review by shehand — APPROVED** [review]
LGTM!

**Review by Niweera — APPROVED** [review]

**Pull request merged**